specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2022-06-01-preview/machineLearningServices.json

```diff
@@ -3662,6 +3713,10 @@
           "x-nullable": true,
           "readOnly": true
         },
+        "idleTimeBeforeShutdown": {
+          "type": "string",
+          "description": "Stops compute instance after user defined period of inactivity. Time is defined in ISO8601 format. Minimum is 15 min, maximum is 3 days."
+        },
```

Comment:
@mentat9  Is this not enough to add the property to Compute Instance PUT?

Reply:
Yes, thanks: I missed this part of the change.

Reply:
@mentat9 This is a new API version, isn't it? Note that my PR is not merging to main, but merging to a branch that will start a new API version. And that branch will go over all ARM validations once everybody has added their changes there. So I believe (and hope) that this individual PR doesn't need to go through the breaking change process.  
The branch this merges to will be looking to create a new API version 2022-06-01-preview

https://dev.azure.com/msdata/Vienna/_workitems/edit/1849890

Reply:
Yes, sorry. Was looking at the submission form that says it's adding features to an existing API version, didn't notice this change was going into an aggregation branch. Been looking at too many of these...